### ISSUE TITLE
Template loading error in 2.1.0 in Commerce Settings

### ISSUE THREAD

**Issue by JayBox325** [craftcms/commerce#751]
### Description
After updating to Craft Commerce 2.1.0 I am now getting Template Not Found error messages at the 'edit' level of all the sections within Settings. As follows:
```
/admin/commerce/settings/producttypes/<id>
```
```
/admin/commerce/settings/gateways/<id>
```

It appears the templates missing for these specific errors are:
```
Unable to find the template “commerce/_layouts/settingscp” in "commerce/settings/producttypes/_edit" at line 1.
```
and
```
Unable to find the template “commerce/_layouts/settingscp” in "commerce/settings/gateways/_edit" at line 1.
```

But it's happening for the all the 'edit' levels within the Settings section.

### Steps to reproduce

1. Try to edit a Gateway
2. Try to edit a Product Type

### Additional info

- Craft version: 3.1.14
- Commerce version: 2.1.0

**Comment by echantigny** [comment]
Same issue with Shipping methods.
ie: /admin/commerce/shipping/shippingmethods/2

**Comment by brandonkelly** [comment]
Thanks. Getting a release out soon with a fix for this…

**Comment by brandonkelly** [comment]
2.1.0.1 is out now with the fix.

**Comment by echantigny** [comment]
Thanks for the ~~quick~~ lightning fast update @brandonkelly !

**Comment by echantigny** [comment]
@brandonkelly Stumbled on some that seems to not be fixed:
Template not found: commerce/settings/taxzones
Template not found: commerce/settings/taxcategories

**Comment by JayBox325** [comment]
Bloody hell, @brandonkelly. Thanks for resolving this so soon! 👍🏼

**Comment by brandonkelly** [comment]
@echantigny Thanks, just released 2.1.0.2 with a fix for those and others.
